### ISSUE TITLE
feat(side by side embedding): UI/UX Stretch Goals

### ISSUE THREAD

**Issue by seve** [chanzuckerberg/single-cell-explorer#947]
### Stretch
- [ ] Move old panels into new side sub-panel
  - [ ] move gene info
  - [ ] move dataset info
    ![image](https://github.com/chanzuckerberg/single-cell-explorer/assets/8716829/f242406e-4ac3-41e8-8482-4fb9b96f0278)

Figma:
https://www.figma.com/design/eJ3acPxG04PKVlom6eS8ft/Spatial?node-id=0-1&t=yAWU6j6wSLKV9eIJ-0

**Comment by seve** [comment]
@kaloster This ticket can be broken out from the side by side work and is unblocked if you have room to add to next sprint
